elliottlib/bzutil.py

```diff
@@ -1115,17 +1115,10 @@ def is_first_fix_any(bugtracker, flaw_bug, current_target_release):
 
     # filter tracker bugs by OCP product
     tracker_bugs = []
-    for tracker_id in tracker_ids:
-        try:
-            b = bugtracker.get_bug(tracker_id)
-        except Exception as e:
-            # first-fix tracker bug might be not visible but not break here
-            if "not authorized" in e:
```

Comment:
at least for current RC, I didn't see not authorized error or other bug can't get error